README.md

```diff
@@ -23,14 +23,15 @@ python3 setup.py install
 ## Configuration
 
 You need to configure a node provider for Linea in your `ape-config.yaml`.
-See `ape-config.yaml.example` 
+See `ape-config.yaml.example`
```

Comment:
Yep, looks like there was a blank space. It's indeed better to avoid unnecessary diff (you'll have to shutdown temporarily your linter plug-in) so the the ApeWorx team only focus on what we we want to change.

Reply:
Ah funny thing 😄 Well, let's keep it then…